processor/src/test/resources/fixtures/org/mapstruct/ap/test/bugs/_913/DomainDtoWithNvmsDefaultMapperImpl.java

```diff
@@ -39,18 +39,15 @@ public Domain create(Dto source) {
         Domain domain = new Domain();
 
         if ( source != null ) {
-            Set<Long> set = stringListToLongSet( source.getStringsInitialized() );
```

Comment:
Nice thing about fixtures is that you can give comments on the generated code :smile: ..Somewhere I made a remark that we should distinguish between: 
* source-null-check, 
* target-null-check and 
* local-var-null-check (or something along that line). 

The latter being the result of a mapping method call or type conversion call. Do we still have that scenario somewhere (e.g. is that `handleLocalVarNullCheck`) ? I guess, if we have a type conversion (after mapping method call)  we still should do null checks and we should also do a source-null-check before we hand the source variable to a type-conversion.

 Ergo should my original  comment be addressed somewhere in the templates or code?